### PR TITLE
Add documentation on running node e2e's with custom image configs

### PR DESCRIPTION
**Pull request by tallclair**
/assign @Random-Liu 
/sig node

### CODE REVIEW

**Comment by tallclair** [comment]
Friendly ping.

File: contributors/devel/sig-node/e2e-node-tests.md [hunk 1]
```diff
@@ -148,6 +148,30 @@ test in parallel against different instances of the same image.
...
+     image_regex: cos-stable-60-9592-84-0
+     project: cos-cloud
+-    metadata: "user-data</go/src/github.com/containerd/cri/test/e2e_node/init.yaml,containerd-configure-sh</go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-extra-init-sh</go/src/github.com/containerd/cri/test/e2e_node/gci-init.sh,containerd-env</workspace/test-infra/jobs/e2e_node/containerd/cri-master/env,gci-update-strategy=update_disabled"
++    metadata: "user-data</home/tallclair/go/src/github.com/containerd/cri/test/e2e_node/init.yaml,containerd-configure-sh</home/tallclair/go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-extra-init-sh</home/tallclair/go/src/github.com/containerd/cri/test/e2e_node/gci-init.sh,containerd-env</home/tallclair/workspace/test-infra/jobs/e2e_node/containerd/cri-master/env,gci-update-strategy=update_disabled"
+```
+
+Finally, run the tests with your custom config:
+
+```sh
+make test-e2e-node IMAGE_CONFIG_FILE="<local file>" [...]
```
Comment (Random-Liu) [inline contributors/devel/sig-node/e2e-node-tests.md]: REMOTE=true?

I think image_config_file only makes sense for remote test.

Comment (tallclair) [reply contributors/devel/sig-node/e2e-node-tests.md]: Thanks! Fixed.

**Comment by Random-Liu** [comment]
@yujuhong For approval

**Pull request merged**